tests/ignite/handlers/test_state_param_scheduler.py

```diff
@@ -0,0 +1,173 @@
+from unittest.mock import patch
+
+import pytest
+import torch
+
+from ignite.engine import Engine, Events
+from ignite.handlers.state_param_scheduler import (
+    ExpStateScheduler,
+    LambdaStateScheduler,
+    LinearStateScheduler,
+    MultiStepStateScheduler,
+    StepStateScheduler,
+)
+
+
+def test_linear_scheduler_linear_increase_history():
+    # Testing linear increase
+    engine = Engine(lambda e, b: None)
+    linear_step_parameter_scheduler = LinearStateScheduler(
+        param_name="linear_scheduled_param",
+        initial_value=0,
+        step_constant=2,
+        step_max_value=5,
+        max_value=10,
+        save_history=True,
+    )
+    linear_step_parameter_scheduler.attach(engine, Events.EPOCH_COMPLETED)
+    engine.run([0] * 8, max_epochs=3)
+    expected_param_history = [0.0, 0.0, 3.3333333333333335]
```

Comment:
Maybe it needs more epochs to improve the validation.

Reply:
yes, will add parametrize tests